CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,8 @@
+# Code of Conduct
+
+As a project hosted by the [Eclipse Foundation](https://github.com/eclipse) on GitHub, we adhere to the 
+[Eclipse Code of Conduct](https://wiki.eclipse.org/Eclipse_Code_of_Conduct) that is based on the
+[Contributor Covenant](https://www.contributor-covenant.org/).
+
+Here we also want to refer to the [Community Code of Conduct](https://github.com/eclipse/.github/blob/master/CODE_OF_CONDUCT.md)
```

Comment:
AFAIK, a `CODE_OF_CONDUCT.md` file in the special `.github` repository of an organization applies to all of the organization's projects / repositories. So do we even need a separate `CODE_OF_CONDUCT.md` file at all?

Reply:
At least the Eclipse's `CODE_OF_CONDUCT.md` is already linked from https://github.com/eclipse/antenna/community.

Reply:
Fair, I'll close it.